concepticondata/conceptrelations.tsv

```diff
@@ -530,3 +529,8 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
 3024	BRIDE PRICE	similar	3168	DOWRY
 3110	CLAP	similar	3154	SLAP
 3171	RINGWORM	similar	3173	TINEA
+2678	HAIR OR FUR	narrower	189	HAIR (BODY)
+3182	MARRIAGE OR WEDDING	narrower	1259	WEDDING
+3182	MARRIAGE OR WEDDING	narrower	2432	MARRIAGE
+3183	BREATHE OR BREATH	narrower	1407	BREATHE
+3183	BREATHE OR BREATH	narrower	870	BREATH
```

Comment:
I am missing the new relations for "life/living" and "dead".

Reply:
You are right, I am going to add those.